sd-rsyslog

```diff
@@ -76,7 +76,8 @@ def onInit():
 
     global process
     if not os.path.exists("/etc/sd-rsyslog.conf"):
-        print("Please create the configuration file at /etc/sd-rsyslog.conf", file=sys.stderr)
+        logging.exception("Please create the configuration file at /etc/sd-rsyslog.conf",
+                          file=sys.stderr)
```

Comment:
```
>>> logging.exception("oops")
ERROR:root:oops
NoneType: None
>>> logging.exception("oops", file=sys.stderr)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/lib/python3.7/logging/__init__.py", line 1969, in exception
    error(msg, *args, exc_info=exc_info, **kwargs)
  File "/usr/lib/python3.7/logging/__init__.py", line 1961, in error
    root.error(msg, *args, **kwargs)
  File "/usr/lib/python3.7/logging/__init__.py", line 1412, in error
    self._log(ERROR, msg, args, **kwargs)
TypeError: _log() got an unexpected keyword argument 'file'
```

The `file=sys.stderr` is leftover from the `print()` invocation, I'll clean up and proceed with re-review.